buildSrc/src/main/kotlin/net/skyscanner/backpack/tokens/BpkColors.kt

```diff
@@ -110,24 +133,21 @@ private fun parseColors(
     return resolveReference(referencing, isDark)
   }
 
-  val list = data
+  return data
     .map {
       BpkColorModel(
         name = it.key.trimName(),
-        defaultValue = it.value.getValue("value").trimColor(),
-        darkValue = it.value["darkValue"]?.trimColor(),
+        category = it.value.getValue("category").removeSuffix("-colors"),
+        isPrivate = it.key.startsWith("PRIVATE_"),
+        defaultValue = it.value.getValue("value"),
```

Comment:
got rid of trimming the color here as we have to modify it later in the process anyways and we no longer want to remove the suffix